package.json

```diff
@@ -2,10 +2,18 @@
   "name": "react-native-x-maps",
```

Comment:
I think we also agreed to rename this to `@Expensify/react-native-maps`

Reply:
I was wondering we can develop the first version with this name and when we're ready to publish, then we can switch to the new name for the npm package

Reply:
if you change the name, a new completely separate package is created in npm.

So I thought we can keep using the existing `react-native-x-maps` and only switch to the official name when your version 1 is ready

Reply:
OK, that works for me 👍